configs/11.0/packages/valgrind/sources

```diff
@@ -20,11 +20,11 @@
 #
 
 ATSRC_PACKAGE_NAME="Valgrind"
-ATSRC_PACKAGE_VER=3.11.0
+ATSRC_PACKAGE_VER=3.13.0
 # Don't update the Valgrind revision without updating the VEX revision too.
 # They have to be in sync.
-ATSRC_PACKAGE_REV=15905
-ATSRC_PACKAGE_VEX_REV=3223
+ATSRC_PACKAGE_REV=16446
+ATSRC_PACKAGE_VEX_REV=3398
```

Comment:
@gftg85 There's another thing. We use ATSRC_PACKAGE_VER=3.13.0 following the pattern MAJOR.MINOR.PATCH and the branches uses the pattern REPOSITORY_MAJOR_MINOR_BRANCH. We can replace dots for underscore but what about the PATCH I should trim it or we can use ATSRC_PACKAGE_VER=3.13 instead?